src/core/ui/icon/BadgeCounter.tsx

```diff
@@ -11,7 +11,7 @@ const BadgeSizes: Record<NonNullable<BadgeCounterProps['size']>, TypographyProps
   small: {
     fontSize: '0.7rem',
     lineHeight: gutters(0.8),
-    width: gutters(0.8),
+    minWidth: gutters(0.8),
```

Comment:
Currently, if you have 10, or 33, or more notifications, the number is cropped as you have a strict circle.
With this change, now if the content expands beyond a single digit, the badge becomes a rectangle instead of circle (rounded).